getting-started/installation.md

```diff
@@ -22,7 +22,7 @@ Press "OK" at the top right. The firmware will upload the Pixhawk and you'll see
 
 <img src="/images/qgc/firmware-3.png" class="img-responsive img-center" />
 
-## Raspberry Pi
```

Comment:
This is unnecessary, no?

Reply:
it will break links that were dropped in the past

Reply:
you are right! I dropped this commit and fixed the link. For some reason I thought anchors were not there by default, or something like that.